### PR TITLE
Feature/timer view

### PR DESCRIPTION
**Pull request by jphorner**
Thank you for pushing your code!
**What (if any) features are you implementing?**
- Added warning message when a category is not selected
- Added helper function to display timer if no errors occur during form submission

**What (if anything) did you refactor?**
- Added new div to the HTML for the category warning since it is separate from the function which evaluates field values
- Invoked timer view function inside the 'else' statement of submitForm()
- Commented out return statement within 'else if' statement to allow multiple field errors to appear

**Were there any issues that arose?**
- The new showTimerView function works fine, but the event listener is attached to the form container
- This means that currently, clicking within the timer view reverts the page to the form view

**Is there anything that you need from your teammate?**
- If time allows, maybe changing 'toggle' in showTimerView to a simple add/remove.hidden method would solve the issue above?

**Any other comments, questions, or concerns?**
- Y'all are awesome. Thanks for working so hard on this.

**Collaborators**

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -79,13 +86,16 @@ function submitForm() {
   event.preventDefault();
   for (var i=0; i < inputFields.length; i++) {
     if (!getCategory()) {
+      categoryWarning.innerHTML = `<img src="./assets/warning.svg" alt="warning sign" id="warning"> A category is required.`;
```
Comment (nzambonivergara) [inline main.js]: Let's remember to make sure that the message does not disappear once user clicks a category button.

File: main.js [hunk 2]
```diff
@@ -100,7 +110,10 @@ function getCategory() {
   return false;
 }
 
-
+function showTimerView() {
```
Comment (nzambonivergara) [inline main.js]: Should we toggle or just add/remove classes?

Comment (nzambonivergara) [inline main.js]: Should we toggle or just add/remove classes?

File: styles.css [hunk 3]
```diff
@@ -22,9 +22,23 @@ main {
   flex-direction: row;
 }
 
+div {
+  padding: 0px;
```
Comment (nzambonivergara) [inline styles.css]: Let's try to keep our units consistent throughout and change our px to rem?

File: styles.css [hunk 4]
```diff
@@ -113,6 +138,7 @@ form > div {
   background-size: 2rem;
   border: .1rem solid #FD8078;
   color: #FD8078;
+  cursor: pointer;
```
Comment (nzambonivergara) [inline styles.css]: Could we put this in all the buttons by using the the button tag selector?

**Comment by nzambonivergara** [comment]
The timer view looks great!! Just added some minor comments for us to remember changes we want to include. Great job! You are awesome!

**Review by nzambonivergara — CHANGES_REQUESTED** [review]
We will continue working on this functionality so we don't want to close the branch yet.

**Comment by jphorner** [comment]
Thanks for the feedback! I'm working with Nolan and we'll make a PR for the latest commits soon.

**Pull request closed without merging**